test/TargetFunctions.sol

```diff
@@ -255,6 +276,34 @@ abstract contract TargetFunctions is Properties {
         // --- Ghost data after ---
         last_action = LastAction.DONATE;
         __totalAssetAfter = woeth.totalAssets();
+        __oeth_balanace_of_woeth = oeth.balanceOf(address(woeth));
+        __hardAssets = woeth.hardAssets();
+        __yieldAssets = woeth.yieldAssets();
+        __yieldEnd = woeth.yieldEnd();
+    }
+
+    /// @notice Handle pass time on chain
+    /// @param _amount Amount of time to pass. 1 Day is maximum, since that is also the 
+    ///        maximum yield time 
+    function handler_pass_time(uint88 _amount) public {
+        uint256 MAX_YIELD_TIME = 1 days;
+
+        // Bound amount of time to pass
+        _amount = uint88(clamp(uint256(_amount), 0, MAX_YIELD_TIME, USE_LOGS));
+        if (_amount == 0) {
+            if (USE_ASSUME) hevm.assume(false);
+            else return;
+        }
```

Comment:
As if `_amount == 0` you return, you can clamp it from 1 instead of 0 and then the assume is not needed.

Reply:
thanks done here: https://github.com/OriginProtocol/woeth-fuzzing-tests/pull/1/commits/3e5cbc756cb883d2ad43c80d17ab955a46de28d5